src/Config.ts

```diff
@@ -64,6 +65,14 @@ export interface ConfigParams {
    * @category Engine
    */
   chooseAddressMappingPolicy: ChooseAddressMapping,
+  /**
+   * Symbol used to denote currency numbers.
+   *
+   * @default '$'
+   *
+   * @category Number
+   */
+  currencySymbol: string,
```

Comment:
GS accepts only one currency symbol at a given moment in the whole spreadsheet.

Reply:
Can you elaborate on that? I see more

<img width="403" alt="Screenshot 2021-01-26 at 19 35 12" src="https://user-images.githubusercontent.com/4718998/105888748-a9877400-600d-11eb-8f37-f3fac5c7a5f3.png">

Reply:
Ok, I see. What do you propose then?

Reply:
This is tricky. Currency is different than Date (which is parsed from a string / on the editor entry). As we did it, parsing "$20" to currency from string, is different than what GS or XL does. They store some kind of metadata with the format. Value is the formula bar is unchanged numeric:

<img width="194" alt="Screenshot 2021-01-27 at 09 56 54" src="https://user-images.githubusercontent.com/4718998/105967376-0d4d8380-6086-11eb-8a62-3eb0570392d4.png">

And entered "$20" is a string. We will have incompatibility here with all the spreadsheets if we keep parsing it to CurrencyType. What we're tracking here is the number formatting, ie. date can have two different formats and they are still passed through the formula:

<img width="374" alt="Screenshot 2021-01-27 at 10 04 36" src="https://user-images.githubusercontent.com/4718998/105968235-1d199780-6087-11eb-8c2d-758353cd4d48.png">

What you did is great to track the Type and set the correct cell editor (number/date/text) in the Handsontable, that is what @jansiegel asked for.

Reply:
Internally should be doable. We can save the format on CurrenctyType and DateType as an object property. And when the type is requested with API both will be available (DateType and DateType.format). This is rather an extension of what we have here.

The problem is how to set the Type and/or Format. Because all we've got is an array of simple types and no metadata: `['1', '1%', '1$', '01/01/1900', '12:00', '01/01/1900 12:00']`. Guess our problem is how to get/set formats from that input. Where:

- `1` is NumericType
- `1%` is PercentType
- `1$` is StringType (for all spreadsheets 💣 )
- `01/01/1900` is DateType, but... `01 01 1900`, `01 Jan 1900`, `01 January 1900` and others are also recognized 🙈

so there is no automatic recognition of the CurrencyType. It's a number with formatting. But take a look at the date type, it's a DateType with different formatting. I guess number, format is also tracked. Try setting scientific at A1 and then add other formats to it `=A1+B1` etc. The first cell (in the formula) format is applied to the result.

Excel has a lot more info, and the CELL function is proof of that: https://support.microsoft.com/en-us/office/cell-function-51bd39a5-f338-4dbe-a33f-955d67c2b2cf with all the properties you can request for a cell.

Reply:
Discussed and resolved in #599.